.github/workflows/checksum.yaml

```diff
@@ -0,0 +1,24 @@
+name: Checksum
+
+on: [push, pull_request]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  ubuntu-build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: true
+    - name: Copy checksums.txt
+      run: cp checksums.txt ci-expected-checksums.txt
```

Comment:
No need to create a copy. It's possible to check whether the file has changed using the following command

```
git diff --exit-code checksums.txt
```